test/scrabble_test.rb

```diff
@@ -5,9 +5,36 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  def test_can_read_values_of_each_letter
+    values = {
+      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
```

Comment:
Pick a couple to test. This gets dangerously close to testing that Ruby itself works, and we want to always test the code that we write.